central/graphql/resolvers/components_v2.go

```diff
@@ -639,6 +639,9 @@ func (eicr *imageComponentResolver) LayerIndex() *int32 {
 
 // PlottedVulns returns the data required by top risky component scatter-plot on vuln mgmt dashboard
 func (eicr *imageComponentResolver) PlottedVulns(ctx context.Context, args RawQuery) (*PlottedVulnerabilitiesResolver, error) {
+	if features.PostgresDatastore.Enabled() {
+		return nil, errors.New("PlottedVulns resolver is not support on postgres. Use PlottedImageVulnerabilities.")
```

Comment:
Should be "... is not supported ..." for this and all other error messages

Reply:
Ah yes, good catch.

I have a follow-up PR anyways. I will address the typo in that PR.